agregar campo de usuario a revision

EduardoCuestaC opened the pull request:
agregar usuario a revisión cuando se hace el posten revisiones de inventario y de estado de ambulancia
mostrar los datos del usuario en lugar de los de la revisión en las views de ver revisiones y ver detalle de revisionespara los dos casos
mostrar turno en ver revisiones y ver detalle revisiones

EduardoCuestaC committed:
agregar campo de usuario a revision
agregar usuario a revisión cuando se hace el posten revisiones de inventario y de estado de ambulancia
mostrar los datos del usuario en lugar de los de la revisión en las views de ver revisiones y ver detalle de revisionespara los dos casos
mostrar turno en ver revisiones y ver detalle revisiones

ReneOv commented on Vostok/ambulancia/views.py:
```diff
@@ -13,6 +13,8 @@
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
 from revision.models import RevisionAmbulancia
+from users.models import CustomUser
```
Cuando importes CustomUser, no utilices ese import. Tiene que ser:

`from django.contrib.auth import get_user_model`

Y despues dentro de la view haces un:
`User = get_user_model()
usuario = User.objects.get(id=id)`

ReneOv commented on Vostok/inventario/views.py:
```diff
@@ -1,4 +1,6 @@
 from django.shortcuts import render, get_object_or_404, redirect, render_to_response
+from users.models import CustomUser
```
x2 del comment pasado.

ReneOv commented on Vostok/revision/models.py:
```diff
@@ -1,20 +1,19 @@
 from django.db import models
 from django.utils import timezone
+from users.models import CustomUser
```
x3 del primer comment

ReneOv reviewed (changes requested):
Ya lo puse en comments directo en "files changed"

tecmanny committed:
Cambio de modelo de ambulancia y ortografica

EduardoCuestaC committed:
cambiar nombre variables checklist

EduardoCuestaC committed:
cambiar nobmres en json de ambulancia

EduardoCuestaC committed:
arreglar import del modelo de usuario

EduardoCuestaC committed:
medidas en api

EduardoCuestaC committed:
medida not null

EduardoCuestaC committed:
agrreglar activaciones de ambulancias y mostrar nombre de las guardias
- reducir número de estados a activa y desactivada
- agregar campo de guardia a usuario
- agregar tabla de activables. se usa para ver cuántas ambulancias
  pueden ser activadasen la app al mismo tiempo
- en la view de control de ambulancia se puede editar el número de
  ambulancias que pueden ser activadas al mismo tiempo

EduardoCuestaC committed:
fix conflicts

EduardoCuestaC committed:
agregar componentes faltantes a view de ver detalle de revision de ambulancia

EduardoCuestaC committed:
Merge branch 'develop' of https://github.com/impulse-semestrei/Vostok into api_ambulancia
